HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -3023,22 +3030,20 @@ def self.add_data_point_adaptive_step_size(data_array, mode, tol = 0.1)
   # TODO
   #
   # @param data_array [TODO] TODO
-  # @param mode [TODO] TODO
+  # @param mode [Symbol] Heating (:htg) or cooling (:clg)
   # @return [nil]
   def self.correct_ft_cap_eir(data_array, mode)
     # Add sensitivity to indoor conditions
     # single speed cutler curve coefficients
     if mode == :clg
-      cap_ft_spec_ss, eir_ft_spec_ss = get_cool_cap_eir_ft_spec(HPXML::HVACCompressorTypeSingleStage)
       rated_t_i = HVAC::AirSourceCoolRatedIWB
-      indoor_t = [50.0, rated_t_i, 80.0]
+      # Added two data points to be held constant outside the range 57F to 72F
+      indoor_t = [40.0, 57.0, rated_t_i, 72.0, 90.0]
```

Comment:
Do we really need to manually hold things constant beyond 57 and 72? When we create the TableLookup object, we set the extrapolation method to "constant" (see [here](https://github.com/NREL/OpenStudio-HPXML/blob/5bdc5548e4ea54640e8c7d438af9a5a3bb4ec35b/HPXMLtoOpenStudio/resources/hvac.rb#L2781)), so wouldn't that already cover this?